adaptive/tests/test_balancing_learner.py

```diff
@@ -21,3 +22,34 @@ def test_balancing_learner_loss_cache():
     bl = BalancingLearner([learner])
     assert bl.loss(real=False) == pending_loss
     assert bl.loss(real=True) == real_loss
+
+
+def test_distribute_first_points_over_learners():
+    for strategy in ['loss', 'loss_improvements', 'npoints']:
+        learners = [Learner1D(lambda x: x, bounds=(-1, 1)) for i in range(10)]
+        learner = BalancingLearner(learners, strategy=strategy)
+        points, _ = learner.ask(100)
```

Comment:
We should also test for a failure mode when e.g. 1 or 3 points are fed into a learner.

Reply:
Done in 6e3e9852e828ee5f5d016f26a4d1e2453c606aaf.